cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,43 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+====================
+Application Logbacks
+====================
+
+YARN containers spawned by a CDAP application use the default container logback, ``logback-container.xml``,
```

Comment:
YARN containers (both master services and user programs) launched by CDAP by default use the logback configuration file present in :ref:`Configuration <install-configuration>` directory —  logback-container.xml.